tools/scripts-config-react-webpack/config/webpack.config.prod.js

```diff
@@ -10,6 +10,7 @@ module.exports = () => ({
 			// This is only used in production mode
 			new TerserPlugin({
 				terserOptions: {
+					sourceMap: true,
```

Comment:
Now `sourceMap` is one of terserOptions properties. https://github.com/terser/terser#minify-options